lib/actions.js

```diff
@@ -218,8 +218,7 @@ actions.saveConfiguration = function(baseName, stackName, stackRegion, bucket, p
 actions.saveTemplate = function(templateUrl, templateBody, callback) {
   var uri = url.parse(templateUrl);
   var prefix = uri.host.replace(/^s3[-.]/, '').split('.');
-  var region = prefix.length === 4 ? 'cn-north-1' :
-    prefix.length === 2 ? 'us-east-1' : prefix[0];
+  var region = prefix.length === 2 ? 'us-east-1' : prefix[0];
```

Comment:
Were you able to validate this against a deploy in cn-north-1? Does that still work as designed?